packages/scripts/src/monitor-dvm/README.md

```diff
@@ -0,0 +1,34 @@
+# DVMv2 Monitoring
+
+## Live Monitor Bots
+
+DVMv2 monitor bots can follow mainnet and goerli networks to report on real time events based on the provided configuration.
+
+The main entry point to live monitor bots is running:
+
+```
+yarn ts-node packages/scripts/src/monitor-dvm/index.ts
+```
```

Comment:
nit: This is just my opinion, but I think we should prefer using tsc + node over ts-node in production:
- tsc is the official typescript implementation built by the typescript team. ts-node is a separate application maintained by a different group as far as I can tell.
- It ensures that all these files are compiled and checked by tsc in our build process. With ts-node, it's possible that we skip building these files entirely in ci or partially build them. If that were the case, we would not get the advantage of typescript checking the types automatically as a part of our development process.
- Performance reasons: see [here](https://stackoverflow.com/questions/60581617/is-it-a-bad-practise-to-use-ts-node-in-production) and [here](https://github.com/TypeStrong/ts-node/issues/104).

IMO, it's totally fine to encourage ts-node for development purposes in the docs, but I think the default should always be tsc + node.

Reply:
agree, fixed